db/index.js

```diff
@@ -0,0 +1,3 @@
+//Create DB Connection
+
+// module.exports = db;
```

Comment:
You can safely delete this - we already have a db folder and structure going. I think this would cause a merge conflict as it is.